src/Helper/Polyfill.php

```diff
@@ -0,0 +1,48 @@
+<?php 
+
+/*
+ * This file is part of the PHP-CLI package.
+ *
+ * (c) Jitendra Adhikari <jiten.adhikary@gmail.com>
+ *     <https://github.com/adhocore>
+ *
+ * Licensed under MIT license.
+ */
+
+namespace Ahc\Cli\Helper;
+
+/**
+ * Polyfill class is for using newer php syntax 
+ * and still maintaining backword compatibility
+ * 
+ * @author  Shlomo Hassid <shlomohassid@gmail.com>
+ * @license MIT
+ *
+ * @link    https://github.com/adhocore/cli
+ */
+class Polyfill
+{
+    public static function str_contains($haystack, $needle)
```

Comment:
is it really needed for php8? what is the target php version for new changes?

Reply:
No, it's not. I generally think new releases should only support 8+,  I thought you were still supporting 7.4 just realized you have a dedicated release for PHP 7.0. 
Mark it in the review I will remove it when you are done with the review 😄.